probe/tag/docker_tagger.go

```diff
@@ -0,0 +1,172 @@
+package tag
```

Comment:
Right, because it was merged today. I'll allocate some time to bring it back tomorrow, if nothing more important comes up.